ketcher-autotests/tests/User-Interface/Clear-Canvas/clear-canvas.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { test, expect, Page } from '@playwright/test';
+import {
+  pressButton,
+  takeEditorScreenshot,
+  openFileAndAddToCanvas,
+  TopPanelButton,
+  selectRing,
+  RingButton,
+  clickInTheMiddleOfTheScreen,
+  selectTopPanelButton,
+  waitForPageInit,
+  waitForRender,
+} from '@utils';
+
+async function addTextBoxToCanvas(page: Page) {
+  await page.getByTitle('Add text (Alt+T)').click();
+  await clickInTheMiddleOfTheScreen(page);
+  await page.getByRole('dialog').getByRole('textbox').click();
+}
```

Comment:
I believe it's better to move this function to utils to reuse it it other tests

Reply:
Done! :)